sssom/util.py

```diff
@@ -1088,12 +1092,22 @@ def get_prefixes_used_in_table(df: pd.DataFrame) -> List[str]:
     """Get a list of prefixes used in CURIEs in key feature columns in a dataframe."""
     prefixes = []
     if not df.empty:
-        for col in KEY_FEATURES:
-            for v in df[col].values:
-                prefixes.append(get_prefix_from_curie(v))
+        for col in ENTITY_REFERENCE_SLOTS:
+            if col in df.columns:
+                for v in df[col].values:
+                    prefixes.append(get_prefix_from_curie(v))
     return list(set(prefixes))
```

Comment:
prefixes should _always_ contain `semapv` and `sssom` prefixes.

Reply:
Should I add "semapv" to [SSSOM_BUILT_IN_PREFIXES](https://github.com/mapping-commons/sssom-py/blob/5490672f3bb8853f0a9a6d01a223a7f476a37f1a/sssom/context.py#L20)?

Reply:
Great!